prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/ConfigSchema.kt

```diff
@@ -222,7 +222,9 @@ abstract class ConfigSchemaElement(
         overwritingElement.schemaRef?.let { this.schemaRef = overwritingElement.schemaRef }
         overwritingElement.resource?.let { this.resource = overwritingElement.resource }
         overwritingElement.resourceIndex?.let { this.resourceIndex = overwritingElement.resourceIndex }
+        if (overwritingElement.debug) this.debug = overwritingElement.debug
```

Comment:
`debug` is a non-nullable Boolean, so this will only overwrite `debug` if it is `true` in the overwriting element. We might instead want this to just be
```suggestion
        this.debug = overwritingElement.debug
```
which would overwrite it when it's false too.

Reply:
That's the key, though. I don't want to overwrite unless it is true in case it is set to true in the parent.